q2_demux/plugin_setup.py

```diff
@@ -135,7 +138,9 @@
               'average positional qualities across all of the sequences '
               'selected. If input sequences are paired end, plots will be '
               'generated for both forward and reverse reads for the same `n` '
-              'sequences.')
+              'sequences.'),
+        'delta': ('The acceptable delta cutoff for disparities in sequence '
+                  'length')
```

Comment:
To me it's more about not crying-wolf in obviously fine situations. So I think it's useful only in so far as it keep users from getting into the habit of ignoring yellow boxes as a rule (although the diversity visualizations may have already taught most that).